generated/src/aws-cpp-sdk-dynamodb/include/aws/dynamodb/DynamoDBClient.h

```diff
@@ -42,13 +43,42 @@ namespace DynamoDB
       Aws::Crt::Variant<smithy::SigV4AuthScheme>,
       DynamoDBEndpointProviderBase,
       smithy::client::JsonOutcomeSerializer,
-      smithy::client::JsonOutcome>,
+      smithy::client::JsonOutcome,
+      Aws::Client::DynamoDBErrorMarshaller>,
     Aws::Client::ClientWithAsyncTemplateMethods<DynamoDBClient>
   {
     public:
       static const char* GetServiceName();
       static const char* GetAllocationTag();
       inline const char* GetServiceClientName() const override { return "DynamoDB"; }
+      
+      DynamoDBClient(const DynamoDBClient& other);
+
+      DynamoDBClient(DynamoDBClient&& other);
+
+      DynamoDBClient& operator=(const DynamoDBClient& other)
+      {
+        if(this == &other) 
+        {
+            return *this;
+        }
+        AwsSmithyClientT::operator=(other);
+        Aws::Client::ClientWithAsyncTemplateMethods<DynamoDBClient>::operator=(other);
+        init(m_clientConfiguration);
+        return *this;
+      }
+
+      DynamoDBClient& operator=(DynamoDBClient&& other)
```

Comment:
do we even need a custom move ctor/assignment operator? thinking that `default` would be enough because the target would be the new owner of the all of the resources, that are already intiialized.

thinking default for move should be enough